plyer/platforms/win/sysinfo.py

```diff
@@ -0,0 +1,64 @@
+import platform
+import os
+import re
+from subprocess import Popen, PIPE
+from plyer.facades import Sysinfo
+from plyer.platforms.win.libs import screenmetric
+
+
+class WindowsSysinfo(Sysinfo):
+
+    values = {}
+
+    def __init__(self, **kwargs):
+        self.SysInfo()
+
+    def SysInfo(self):
+
+        cache = os.popen2("SYSTEMINFO")
```

Comment:
Should I create a thread here? because application takes noticable amout of time to load because of this.